FlashCards.AhmadJer99/FlashCards/Dtos/CardDto.cs

```diff
@@ -0,0 +1,9 @@
+﻿
+namespace FlashCards.Dtos;
+
+internal class CardDto
+{
+    public int cardnumber { get; set; }
```

Comment:
🟠 **Naming Conventions**

💡 Properties should be PascalCase.

🔧 `Front`, `Back`

Reply:
Done fixed the properties names